its/ruling/src/integrationTest/java/org/sonarsource/iac/IacRulingTest.java

```diff
@@ -81,6 +81,8 @@ public static void setUp() throws IOException {
     var languageProfile = ProfileGenerator.generateProfile(orchestrator.getServer().getUrl(), "java", "javaconfig", languageRulesConfiguration, Collections.emptySet());
     orchestrator.getServer().restoreProfile(FileLocation.of(languageProfile));
 
+    orchestrator.getServer().restoreProfile(FileLocation.of("src/integrationTest/resources/java-rules.xml"));
```

Comment:
Please add `java` to `LANGUAGES` and remove lines 80-84. 
Thanks to that we don't need to extend `java-rules.xml` each time we add a new rule.

Reply:
Well I thought about that, but I'm not sure we should do it.
This profile only contains the S1135 and S2260, we don't expect to raise on other java rules soon, as spring config rules are not affected here.

If we would add `java` to the `LANGUAGES` we would analyze all java files with all java rules (because of the java plugin), so we would need to exclude  *.java files for every project analyzed in the Ruling Test.

Reply:
1.) These rules are from the javaconfig repository, not the java repository. They are activated by line 80 to 82, not line 84.

2.) This only applies to the spring-config project. But the change you suggested would required us to make this exclusion for every project

Reply:
Adding java to the languages would not properly register the javaconfig rules. I will add a comment to make it more clear how it's working.

Reply:
And yes, right now it has no real meaning because there are no javaconfig rules. But this way we don't have to remember later to add it.

Reply:
Added some comments to clarify the behavior.